Jenkinsfile

```diff
@@ -99,5 +101,26 @@ pipeline {
         """
       junit testResults: "logs/junit.xml", keepLongStdio: false
     }
+    success {
+      script {
+        if( env.UMB_MESSAGE == true || env.UMB_MESSAGE == "true" ){
+          sendCIMessage \
+            providerName: 'Red Hat UMB', \
+            overrides: [ topic: 'VirtualTopic.qe.ci.jenkins' ], \
+            failOnError: false, \
+            messageType: 'Tier1TestingDone', \
+            messageProperties: '''
+              TOOL=ocs-ci
+              PRODUCT=ocs
```

Comment:
in the RHV we used to have this jms selector for the trigger job:
CI_TYPE = 'product-build-done' AND PROJECT = 'RHV-4.3-CI-RELEASE'

So we selected just for the specific version defined in PROJECT message property.  If you have defined ocs in PRODUCT we will need to have the -$version  like ocs-4.2 or not sure if there can be another message property like VERSION where we can specify this and have the selector set for specific version.

Reply:
There is a separate PRODUCT_VERSION field that I've seen used elsewhere

Reply:
I am fine with that as well to see there now version 4.2.

Reply:
will you add this PRODUCT_VERSION as we discussed here?